### PR TITLE
[EN] OZT2018

### PR DESCRIPTION
**Pull request by BearZyo**
### Description
Began the OZT2018 wiki page, by adding a title, headings, and information within each of them, and fixing some errors.

### Status
Pending Review

### Additional Notes 
New PR, from a previous closed PR ([EN] OZT2018), to squash commits.

### CODE REVIEW

File: wiki/Tournaments/OZT/2018/en.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+![osu!Z-Tournament 2018](OZT2018.png)
+
+The **osu! Z-Tournament 2018**, is a 4v4 international osu!standard tournament, hosted by BearZyo and Div. This is the 5th iteration of the Z-Tournament series; the previous iteration could be found [**here**](https://osu.ppy.sh/forum/t/580366).
+
+## Links
+- [OZT2018 Discord server](https://discord.gg/29RmHDQ)
+- [OZT2018 Twitch stream](https://www.twitch.tv/ztournament)
+- [OZT2018 Spreadsheet](https://docs.google.com/spreadsheets/d/e/2PACX-1vT24OMXj6iKV2XeAHADas4JtTGi1E_dAfxeiAMt4-j_smF62yWz92H03LUrLkLjltk9M6zHz7iQSHlk/pubhtml)
+- [OZT2018 Ruleset](https://docs.google.com/document/d/1JailsFw7ZMr9svB_gxxd0uMX2yCkhw7CY4VX4M4TTQs/edit?usp=sharing)
+- OZT2018 Map pool
```
Comment (fajar13k) [inline wiki/Tournaments/OZT/2018/en.md]: Does this suppose to have link?

Comment (BearZyo) [reply wiki/Tournaments/OZT/2018/en.md]: It's fine, I'll just add a link

File: wiki/Tournaments/OZT/2018/en.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ Be aware that becoming a free agent will **not guarantee a spot for playing in t
...
+| Referees | [![][flag_US] BearZyo](https://osu.ppy.sh/users/6116759), [![][flag_NZ] Div](https://osu.ppy.sh/users/3751116), [![][flag_PS] Omaru](https://osu.ppy.sh/users/6574143), [![][flag_CA] Sparhten](https://osu.ppy.sh/users/7601720), [![][flag_CA] Sakaguchi](https://osu.ppy.sh/users/7057311), [![][flag_US] Science](https://osu.ppy.sh/users/5535233), [![][flag_US] motive](https://osu.ppy.sh/users/7376116), [![][flag_GB] Yazzehh](https://osu.ppy.sh/users/7068973), [![][flag_GB] AlexDark69](https://osu.ppy.sh/users/5146647), [![][flag_US] divinereigns](https://osu.ppy.sh/users/9244014), [![][flag_FR] Jean-Michel Jr](https://osu.ppy.sh/users/4430948), [![][flag_LT] Mancis](https://osu.ppy.sh/users/4839300), [![][flag_NL] Mr HeliX](https://osu.ppy.sh/users/2330619), [![][flag_SE] Nitroz](https://osu.ppy.sh/users/5256529), [![][flag_US] BlackZero](https://osu.ppy.sh/users/5383496), [![][flag_DE] Xenou](https://osu.ppy.sh/users/2539403),  [![][flag_US] gummimallow](https://osu.ppy.sh/users/6372755), [![][flag_US] _p0ke_](https://osu.ppy.sh/u/5434711) |
```
Comment (MegaApplePi) [inline wiki/Tournaments/OZT/2018/en.md]:
```diff
- [![][flag_US] _p0ke_](https://osu.ppy.sh/u/5434711)
+ [![][flag_US] \_p0ke\_](https://osu.ppy.sh/users/5434711)
```
escape the underscores in the username and change `/u/` to `/users/`

File: wiki/Tournaments/OZT/2018/en.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+## Staff 
+| Role | Member(s) |
+| ---: | :--- |
+| Host | ![][flag_US] [BearZyo](https://osu.ppy.sh/users/6116759) |
+| Co-Host | ![][flag_NZ] [Div](https://osu.ppy.sh/users/3751116) |
+| Spreadsheet management | ![][flag_CA] [Meow Mix](https://osu.ppy.sh/users/3021634)
+| Statistician | ![][flag_US] [BearZyo](https://osu.ppy.sh/users/6116759) |
+| Programmer | ![][flag_KR] [Eurus](https://osu.ppy.sh/users/3426414) |
+| Media | ![][flag_US] [Weed](https://osu.ppy.sh/users/2609349) |
+| Graphics | ![][flag_FI] [-Bye](https://osu.ppy.sh/users/4291882) |
```
Comment (Druwaid) [inline wiki/Tournaments/OZT/2018/en.md]: -Bye should have a Swedish flag

**Pull request merged**